src/ui/public/vislib/visualizations/_map.js

```diff
@@ -47,16 +51,15 @@ export default function MapFactory(Private) {
     this._valueFormatter = params.valueFormatter || _.identity;
     this._tooltipFormatter = params.tooltipFormatter || _.identity;
     this._geoJson = _.get(this._chartData, 'geoJson');
-    this._mapZoom = params.zoom || defaultMapZoom;
+    this._mapZoom =  Math.max(Math.min(params.zoom || defaultMapZoom, tilemapOptions.maxZoom), tilemapOptions.minZoom);
     this._mapCenter = params.center || defaultMapCenter;
     this._attr = params.attr || {};
 
     let mapOptions = {
-      minZoom: 1,
-      maxZoom: 18,
+      minZoom: tilemapOptions.minZoom,
+      maxZoom: tilemapOptions.maxZoom,
       noWrap: true,
       maxBounds: L.latLngBounds([-90, -220], [90, 220]),
-      scrollWheelZoom: false,
```

Comment:
Agreed @epixa. I created https://github.com/elastic/kibana/issues/7778 to make that happen once this is committed.